lib/ret/media_search.ex

```diff
@@ -486,14 +486,25 @@ defmodule Ret.MediaSearch do
     {:commit, results}
   end
 
-  defp favorites_search(cursor, _type, account_id, _query, order \\ [desc: :last_activated_at]) do
+  defp rooms_must_allow_entry(results, "rooms") do
```

Comment:
Was a bit confused by the need for differentiation on different types, since only rooms can be favorites, but we carry through that assumption through the rest of the flow so this makes sense (though it should be noted that the actual thing doing the rendering `favorite_to_entry` does not)